docs/sdks/dotnet/GETTING_STARTED.md

```diff
@@ -1,44 +1,47 @@
 ## Getting Started
 
 ### Initialize & Make API Request
-Once you add the dependencies, its extremely easy to get started with the SDK; All you need to do is import the package in your code, set your Appwrite credentials, and start making API calls. Below is a simple example:
+Once you have installed the package, it is extremely easy to get started with the SDK; all you need to do is import the package in your code, set your Appwrite credentials, and start making API calls. Below is a simple example:
 
 ```csharp
 using Appwrite;
+using Appwrite.Services;
+using Appwrite.Models;
 
-static async Task Main(string[] args)
-{
-  var client = Client();
+var client = new Client()
+  .SetEndpoint("http://cloud.appwrite.io/v1")  // Make sure your endpoint is accessible
```

Comment:
You don't need to make sure endpoint is accessible anymore ;) It's cloud

Reply:
Might still be a good note for people who replace it with self-hosted links :)